includes/utilities.inc

```diff
@@ -426,6 +426,52 @@ function islandora_paged_content_pdf_combine(array $files, $out) {
 }
 
 /**
+ * Appends a series of OCR files to a consolidated OCR file.
+ *
+ * @param string $consolidated_ocr
+ *   The consolidated OCR path.
+ * @param array $files
+ *   The files to be appended to the OCR.
+ *
+ * @return bool
+ *   TRUE on success, FALSE on failure.
+ */
+function islandora_paged_content_ocr_append($consolidated_ocr, array $files) {
+  $temp_file = drupal_tempnam("temporary://$consolidated_ocr.temp.txt", 'tmp');
```

Comment:
Not quite sure this'll do what you expect? [`drupal_tempnam()`](https://api.drupal.org/api/drupal/includes!file.inc/function/drupal_tempnam/7)'s first parameter is the directory in which to create the file... I'm not sure it'll actually create missing directories...